### PR TITLE
HHH-9469 Windows platform build fixes

### PR DESCRIPTION
**Pull request by dlmiles**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This file is used by unit tests and data I/O testing the CRLF/LF
+#  mode of the checked out working tree is important to ensure the unit
+#  test passes.
+hibernate-core/src/test/resources/org/hibernate/test/fileimport/multi-line-statements.sql eol=lf text
```
Comment (brmeyer) [inline .gitattributes]: Would it be better to try and use a wildcard here?  Maybe something like the following?
`**/test/resources/**/*.sql eol-lf text`

If I remember correctly, git isn't able to automatically filter out binary files, so explicit file extensions are needed.  .sql is a good start.

**Comment by dlmiles** [comment]
This was done ages ago when I was unit testing with Windows(Intellij/Eclipse) and  PostgreSQL(running over network on Linux).

Maybe there are other files added since the date of the patch to the tree, but I try to just do the minimum needed to fix the unit test.  If you know more about the status of the HIB project that a wildcard would be better.

Also it is `eof=lf` you are setting the mode of the 'eof' parameter.

Are you looking for an updated patch ?  I need to checkout tree again, and retest, etc... could take a few weeks to sort (I am not a HIB developer).

**Comment by vladmihalcea** [comment]
Thanks for the Pull Request. Did you sign the [CLA](https://cla.jboss.org/)?

**Comment by richardfontana** [comment]
@vladmihalcea these can be merged without needing the CLA.

**Comment by vladmihalcea** [comment]
I have a Windows machine and even without these changes all tests are running just fine now.

**Comment by dlmiles** [comment]
What is your EOL setting for git on windows.  So maybe they work for you but still do not work for others.

So please check the tests they related to have changed to not be fussy over the EOL conventions of the sample output data.  This way it does not matter what the EOL is.  Or manually change the EOL of the marked files to the other kind of EOL and rerun the tests.

**Comment by vladmihalcea** [comment]
I have `core.autocrlf=true` and everything works just fine. Are you able to replicate the test failure on your machine?

**Pull request closed without merging**